subschemas/substrate-chain/src/resolvers/Query/parachains.ts

```diff
@@ -22,10 +26,60 @@ export async function parachainsInfo(
   };
 }
 
-export function parachains(_: Record<string, never>, __: Record<string, never>, {api}: Context) {
-  throw new Error('Not implemented yet');
+export async function parachains(_: Record<string, never>, __: Record<string, never>, {api}: Context): Promise<Promise<Parachain>[]> {
+  const [parachainIds, genesisHash, lastEvents] = await Promise.all([
+    api.query.paras?.parachains?.<ParaId[]>(),
+    api.genesisHash.toHex(),
+    getLastEvents(api),
+  ]);
+
+  const startingEndpoints = createWsEndpoints((key: string, value: string | undefined) => value || key);
+  const endpoints = startingEndpoints.filter(({genesisHashRelay}) => genesisHash === genesisHashRelay);
+
+  const parachains = parachainIds.map(paraId => {
+    const parachain = endpoints.find((e) => e.paraId === paraId.toNumber());
+
+    if (!parachain) {
+      return undefined;
```

Comment:
We should not filter out parachains/parathreads that are not in the `apps-config`, it could be that the `package` (dependency version) that we have installed is outdated, remember that currently we have to manually update the npm dependency. The approach should be if we cannot find a `paraId` in the endpoints then the homepage, project name, etc should be null but we should still return the rest of the data available on chain.